data/json/items/generic.json

```diff
@@ -2153,6 +2153,57 @@
     "volume": 4,
     "bashing": 4
   },
+  {
+    "type": "GENERIC",
+    "id": "soft_adobe_brick",
+    "category": "spare_parts",
+    "symbol": ",",
+    "color": "brown",
+    "name": "soft adobe brick",
+    "name_plural": "soft adobe bricks",
```

Comment:
This line is redundant as system automatically adds `s` to the end of the `name` if no `name_plural` is set.